### DISCUSSION TITLE
Controls for Row Ordering

### DISCUSSION THREAD

**Discussion by thescripted** [TanStack/table#5724] [Q&A] [answered]
Hello & Thanks for the work you've all contributed to the library,

Does Tanstack Table provide a way to reorder rows, similar to the Column Ordering APIs? If not, how should I go about extending TanStack to enable this feature through the existing APIs? Ultimate goal is to provide controls to users to re-order their rows via drag and drop. 

Unlike Column Ordering, I'm aware of some of the limitation and "pitfalls" that Row Ordering can cause, such as out-of-sync sorting. I'm okay with managing the complexity to deal with this on my end, but would like some guidance on how this feature can be extended from TanStack.

Thanks!

**Comment by KevinVandy** [comment, marked as answer]
https://tanstack.com/table/latest/docs/framework/react/examples/row-dnd

**Reply by thescripted** [reply]
Thanks, I double checked the docs and missed this twice😓

**Reply by iamrakhmatov** [reply]
example in the docs is not working
